libsolidity/inlineasm/AsmScope.h

```diff
@@ -85,6 +85,7 @@ struct Scope
 		Function(std::vector<JuliaType> const& _arguments, std::vector<JuliaType> const& _returns): arguments(_arguments), returns(_returns) {}
 		std::vector<JuliaType> arguments;
 		std::vector<JuliaType> returns;
+		boost::optional<LabelID> id;
```

Comment:
Not sure I like that seemingly backend specific data is in the analyzer. Webassembly will require different kind of data, which wouldn't make sense cluttering with the generic codebase.